tenable/io/v3/base/endpoints/explore.py

```diff
@@ -77,28 +119,17 @@ def search(
                 If ``return_json`` was set to ``True``, then a response
                 object is instead returned instead of an iterable.
 
-        Examples:
-
-            >>>
```

Comment:
no example

Reply:
we have added the examples in the endpoints search function.
since this is not directly accessed by the user and having the similar representation on both, so we can avoid this.
 https://github.com/dhvanilshah-tenable/pyTenable/pull/12/files/5ad8d493343c0bdb08abdf5a755b7c154b5ec1d3#:~:text=of%20an%20iterable.-,Examples%3A,%27%27%27,-iclass%20%3D%20AssetSearchIterator